src/devices/Pn532/README.md

```diff
@@ -0,0 +1,212 @@
+	mifareCard.KeyA = new byte[6] { 0x00, 0x00, 0x00, 0x00, 0x00, 0x00 };
+	mifareCard.KeyB = new byte[6] { 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF };/
+	for (byte block = 0; block < 64; block++)
+	{
+		mifareCard.BlockNumber = block;
+		mifareCard.Command = MifareCardCommand.AuthenticationB;
+		var ret = mifareCard.RunMifiCardCommand();
+		if (ret < 0)
+		{
+			// Try another one
+			mifareCard.Command = MifareCardCommand.AuthenticationA;
+			ret = mifareCard.RunMifiCardCommand();
+		}		
+		if (ret >= 0)
+		{
+			mifareCard.BlockNumber = block;
+			mifareCard.Command = MifareCardCommand.Read16Bytes;
+			ret = mifareCard.RunMifiCardCommand();
+			if (ret >= 0)
+				Console.WriteLine($"Bloc: {block}, Data: {BitConverter.ToString(mifareCard.Data)}");
+			else
+			{
+				Console.WriteLine($"Error reading bloc: {block}, Data: {BitConverter.ToString(mifareCard.Data)}");
+			}
+			if (block % 4 == 3)
+			{
+				// Check what are the permissions
+				for (byte j = 3; j > 0; j--)
+				{
+					var access = mifareCard.BlockAccess((byte)(block - j), mifareCard.Data);
+					Console.WriteLine($"Bloc: {block - j}, Access: {access}");
+				}
+				var sector = mifareCard.SectorTailerAccess(block, mifareCard.Data);
+				Console.WriteLine($"Bloc: {block}, Access: {sector}");
+			}
+		}
+		else
+		{
+			Console.WriteLine($"Autentication error");
+
+		}
+	}
+```
+
+## PN532 as a target
+
+It's possible to change the PN532 mode to be seen as a target byt another reader. A phone with NFC for example. The bellow example shows how to transform the PN532 into a Credit Card:
+
+```csharp
+static void AsTarget(Pn532 pn532)
+{
+	byte[] retData = null;
+	TargetModeInitialized modeInitialized = null;
+	while ((!Console.KeyAvailable))
+	{
+		(modeInitialized, retData) = pn532.InitAsTarget(
+			TargetModeInitialization.PiccOnly, 
+			new TargetMifareParameters() { Atqa = new byte[] { 0x08, 0x00 }, Sak = 0x60 },
+			new TargetFeliCaParameters() { NfcId2 = new byte[] { 0x01, 0xFE, 0xA2, 0xA3, 0xA4, 0xA5, 0xA6, 0xA7 }, Pad = new byte[] { 0xC0, 0xC1, 0xC2, 0xC3, 0xC4, 0xC5, 0xC6, 0xC7 } },
+			new TargetPiccParameters() { NfcId3 = new byte[] { 0xAA, 0x99, 0x88, 0x77, 0x66, 0x55, 0x44, 0x33, 0x22, 0x11 }, GeneralTarget = new byte[0], HistoricalTarget = new byte[0] });
+		if (modeInitialized != null)
+			break;
+		// Give time to PN532 to process
+		Thread.Sleep(200);
+	}
+	if (modeInitialized == null)
+		return;
+
+	Console.WriteLine($"PN532 as a target: ISDep: {modeInitialized.IsDep}, IsPicc {modeInitialized.IsISO14443_4Picc}, {modeInitialized.TargetBaudRate}, {modeInitialized.TargetFramingType}");
+	Console.WriteLine($"Initiator: {BitConverter.ToString(retData)}");
+	// 25-D4-00-E8-11-6A-0A-69-1C-46-5D-2D-7C-00-00-00-32-46-66-6D-01-01-12-02-02-07-FF-03-02-00-13-04-01-64-07-01-03
+	// 11-D4-00-01-FE-A2-A3-A4-A5-A6-A7-00-00-00-00-00-30            
+	// E0-80
+
+	Span<byte> read = stackalloc byte[512];
+	int ret = -1;
+	while (ret<0)
+		ret = pn532.ReadDataAsTarget(read);
```

Comment:
indentation is off

Reply:
seems not just here - double check this whole file

Reply:
hum, that's weird, all look all ok. It's a regular tab generated by Visual Studio like in any other file. With the all renames, I'll have anyway to replace most of the code from the sample file. So let's see how it will look like.